src/Pages/CategoryPage.tsx

```diff
@@ -9,17 +9,30 @@ function CategoryPage() {
   const { name } = useParams();
   const { items, categories, fetchCategoriesFromDb, fetchItemsFromDb } =
     useItems();
+  const [itemsFilterState, setItemsFilterState] = useState<any>("all");
```

Comment:
Why the choise to use a string and not a boolean? :)

Reply:
I did not think about it but i can change it

Reply:
it is updated now